src/session.c

```diff
@@ -1045,56 +1143,21 @@ LINK_ENDPOINT_HANDLE session_create_link_endpoint(SESSION_HANDLE session, const
     return result;
 }
 
-void session_destroy_link_endpoint(LINK_ENDPOINT_HANDLE link_endpoint)
+void session_detach_link_endpoint(LINK_ENDPOINT_HANDLE link_endpoint)
```

Comment:
>session_detach_link_endpoint [](start = 5, length = 28)

The poetry of this causes asymmetry.  You now create a session link and detach a session link instead of create a session link and destroy a session link.  It's just curious why you decided to go this direction.

Reply:
You know, you are right. I'll simply leave this be destroy and make the destroy be an internal free.
Cheers.

---
In reply to: [189097822](https://github.com/Azure/azure-uamqp-c/pull/236#discussion_r189097822) [](ancestors = 189097822)